File: 1281-subtract-the-product-and-sum-of-digits-of-an-integer/1281-subtract-the-product-and-sum-of-digits-of-an-integer.js
/**
 * @param {number} n
 * @return {number}
 */
var subtractProductAndSum = function(n) {
    let multiply = 1
    let total = 0
    let stringN = String(n)
    for(let i = 0; i < stringN.length; i++){
        multiply *= parseInt(stringN[i])
        total += parseInt(stringN[i])
    }
    return multiply - total
};

File: 1281-subtract-the-product-and-sum-of-digits-of-an-integer/1281-subtract-the-product-and-sum-of-digits-of-an-integer.java
class Solution {
    public int subtractProductAndSum(int n) {
        int total = 0;
        int multiply = 1;
        while (n > 0){
            multiply *= n % 10;
            total += n % 10;
            n = n / 10;
        }
        return multiply - total;   
    }
}